packages/hardhat-chai-matchers/package.json

```diff
@@ -19,6 +19,7 @@
     "lint:fix": "yarn prettier --write && yarn eslint --fix",
     "eslint": "eslint 'src/**/*.ts' 'test/**/*.ts'",
     "prettier": "prettier \"**/*.{js,md,json}\"",
+    "pretest": "cd ../.. && yarn build",
     "test": "mocha --recursive \"test/**/*.ts\" --exit",
     "test:ci": "yarn test && node scripts/check-subpath-exports.js",
     "build": "tsc --build .",
```

Comment:
I don't see the benefit, but I'm open to being convinced. Apart from tests, could you provide an example of a situation (during package development) where you'd prefer to build everything instead of just the package?

Reply:
It was just about consistency, like "every time we build we just build everything to be safe". But it's a super weak preference, feel free to merge as it is.